File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Admin/AdminUserUpdateResponseDto.ts
// src/app/models/AdminUserUpdateResponseDto.ts
export interface AdminUserUpdateResponseDto {
    password: string;
    name: string;
    lastname: string;
    email: string;
    bank: string;
    cbu: string;
    cuil: string;
    phone: number;
    deleted: boolean;
    blocked: boolean;
    commentBlocked: string;
    rol: string;
  }
  

File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/admin/AdminUserUpdateResponseDto.java
package ps.com.viajeros.dtos.admin;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class AdminUserUpdateResponseDto  {
    private String password;
    private String name;
    private String lastname;
    private String email;
    private String bank;
    private String cbu;
    private String cuil;
    private Long phone;
    private boolean deleted;
    private Boolean blocked;
    private String commentBlocked;
    private String rol;
}